services/notifications.go

```diff
@@ -639,21 +639,6 @@ func queueEmailNotifications(notificationsByUserID map[uint64]map[types.EventNam
 			continue
 		}
 		go func(userEmail string, userNotifications map[types.EventName][]types.Notification) {
-			notification := ""
-			othernotifications := ""
-			i := 0
-			for notificationEvent := range userNotifications {
-				if i == 0 {
-					notification = string(notificationEvent)
-				} else if i == 1 {
-					othernotifications = fmt.Sprintf(" and %s", notificationEvent)
-				}
-				i++
-			}
-			if i > 1 {
-				othernotifications = fmt.Sprintf(",... and %d other notifications", i)
```

Comment:
The second here, it is weird to say "and 5 others" if it is 5 notifications in total.